apps/electron-app/src/main/services/agent/agent-config.ts

```
@@ -0,0 +1,133 @@
+  static mergeWithDefaults(partialConfig: Partial<AgentConfig>): AgentConfig {
+    if (!partialConfig.openaiApiKey) {
+      throw new Error("OpenAI API key is required");
+    }
+
+    return {
+      ...DEFAULT_AGENT_CONFIG,
+      ...partialConfig,
+      openaiApiKey: partialConfig.openaiApiKey.trim(),
+    };
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)